polkadot-protocol/protocol-components/overview.md

```diff
@@ -0,0 +1,253 @@
+
+When a node creates or receives a new block, it must "gossip" (broadcast) this block to other nodes in the network. 
+
+Other nodes will track these announcements and can request more information about the blocks. 
+
+This mechanism is crucial for maintaining consensus and ensuring all nodes have the most current state of the blockchain.
+
+!!!note
+    For more detailed information on how blocks are announced please refer to [Polkadot Spec](https://spec.polkadot.network/chap-networking#sect-msg-block-announce){target=\_blank} documentation 
+
+### Extrinsics
+
+An extrinsic is a [SCALE](https://github.com/paritytech/parity-scale-codec#parity-scale-codec){target=\_blank} encoded array consisting of a `version number`, `signature`, and varying `data` types indicating the resulting runtime function to be called, including the parameters required for that function to be executed.
+
+Extrinsics constitute information from the outside world and take on three forms:
+
+- Inherents
+- Signed Transactions
+- Unsigned Transactions
+
+As an infrastructure provider, you will deal almost exclusively with signed transactions. You will, however, see other extrinsics within the blocks that you decode as well. 
+
+#### Inherents
+
+Inherent extrinsics are unsigned transactions that contain information based on validators' consensus rather than verifiable truth. 
+
+For example, while a timestamp can't be definitively proven, validators can agree it falls within a reasonable range according to their system clocks. 
+
+Unlike standard transactions, inherent extrinsics are included in blocks produced by validators and are not shared individually throughout the network.
+
+#### Signed Transaction
+
+Signed transactions contain a signature from the account that initiated the transaction. 
+
+This signature verifies the authenticity of the transaction and indicates that the account will pay a fee for its inclusion on the blockchain. 
+
+Because the potential value of signed transactions is identifiable before execution, they can be shared across the network with minimal risk of spam, making them suitable for efficient node communication.
+
+This concept aligns with how transactions function in Ethereum and Bitcoin, where signed transactions are essential for validating and processing operations on the blockchain.
+
+#### Unsigned Transactions
+
+Some transactions require unsigned transactions because they cannot be initiated by an account with sufficient funds to pay fees. 
+
+For instance, when a user claims DOT tokens from an Ethereum DOT indicator contract to a new DOT address, the new address starts with zero funds and cannot cover transaction fees. 
+
+Thus, unsigned transactions are essential in enabling actions in such scenarios, ensuring users can participate without funding limitations.
```

Comment:
This entire section overlaps with: https://github.com/polkadot-developers/polkadot-docs/pull/77/files#diff-0ab151ab7f6f1f93b152f9f43576a07577693fb69d7f117072556994daed5c17R12

I'd leave only the extrinsic paragraph here and then maybe add a link to the docs of transactions for further details.